### PR TITLE
Release290

### PR DESCRIPTION
**Pull request by aindriu-aiven**
Adding Release 2.9.0 release notes.
Also adding some information and caveats around BETA feature claim subscriptions

### CODE REVIEW

File: docs/releases/release290.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+---
+sidebar_position: 99.87
```
Comment (muralibasani) [inline docs/releases/release290.md:2]:
```suggestion
sidebar_position: 99.86
```

Comment (muralibasani) [reply docs/releases/release290.md:2]: to keep it on the top

File: docs/releases/release290.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+---
+
+# Release 2.9.0
+
+Date: 8th of April, 2024
+
+## Overview
+
+Overview
+Klaw version 2.9.0 is a patch release which fixes an issue introduced in 2.9.0 with some minor bug fixes and improvements.
```
Comment (muralibasani) [inline docs/releases/release290.md:12]:
```suggestion
Klaw version 2.9.0 is a patch release which fixes an issue introduced in 2.8.0 with some minor bug fixes and improvements.
```

**Review by muralibasani — APPROVED** [review]
LGTM, thx

File: docs/workflows/subscription/claim-subscription.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+1. **Topic owner approval (data owner)**: The topic (data) owner, as the data owner, must approve the request first.
+2. **Existing owner approval (subscription transfer)**: The current subscription owner must approve the transfer to the new team.
+
+The **Service User** associated with the subscription won't be transferred until **all** subscriptions owned by the original team for that user are transferred to the new team.
+
+:::note
+While the UI might not explicitly state it, the topic owner's approval is always required first. We're actively working on improving the UI clarity in future updates.
+:::
+
+## Create a claim subscription request
```
Comment (aindriu-aiven) [inline docs/workflows/subscription/claim-subscription.md:24]: @harshini-rangaswamy should this be the heading for line 9 and vice versa?

Comment (harshini-rangaswamy) [reply docs/workflows/subscription/claim-subscription.md:24]: The heading here is good; it aligns with the process of creating a subscription request. 

I moved the "About claim subscription" section ahead of the How-to section to provide more contextual information. Let me know if it's not clear and we can move it back.

**Review by muralibasani — APPROVED** [review]

**Pull request merged**